contracts/staking/query.go

```diff
@@ -77,3 +76,59 @@ func QueryValidators(t TxQueryHandler, from types.Address) ([]types.Address, err
 
 	return DecodeValidators(method, res.ReturnValue)
 }
+
+func DecodeBLSPublicKeys(
+	method *abi.Method,
+	returnValue []byte,
+) ([][]byte, error) {
+	decodedResults, err := method.Outputs.Decode(returnValue)
+	if err != nil {
+		return nil, err
+	}
+
+	results, ok := decodedResults.(map[string]interface{})
+	if !ok {
+		return nil, errors.New("failed type assertion from decodedResults to map")
+	}
```

Comment:
This part is exactly the same as `DecodeValidators`

Reply:
Created a helper function at 83075ece805ef094ff99bb271259528fe9a52017